### DISCUSSION TITLE
User Readme doesn’t display everything

### DISCUSSION THREAD

**Discussion by BenjaminMichaelis** [community/community#38049] [Mobile] [answered]
The readme from a user profile page and when you click on the readme itself are different and the user one doesn't display all that the view when you click on a readme does and displays a question mark box instead. Screenshots and video attached.

![image](https://user-images.githubusercontent.com/22186029/199422923-1915f796-6c59-48f6-95a3-3451d18bd842.png)

![image](https://user-images.githubusercontent.com/22186029/199422951-03be3e07-a5fa-4e87-9af0-a66562f95f41.png)

https://user-images.githubusercontent.com/22186029/199423121-835c64eb-eb88-46ea-a31f-c39a56d00d46.MOV

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.86.0 (81658795)
Device: iPhone 13 Pro (iOS 16.0)
</details>

**Comment by david-engelmann** [comment]
It appears that it is working correctly on Android and the Desktop version. Do you have the most up to date version of the Github for iOS?

**Reply by BenjaminMichaelis** [reply]
Yes

**Comment by stevepopovich** [comment, marked as answer]
Hi @BenjaminMichaelis, thanks for reaching out. I will file an issue with the team. Thanks!
